### PR TITLE
Add blocked and stopped hooks

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: hooks/hooks_test.go [hunk 1]
```diff
@@ -225,3 +231,61 @@ func RunActionTestCases(t *testing.T, tcs []HookTestCase) {
 		rc.Close()
 	}
 }
+
+func RunEventsTestCases(t *testing.T, tcs []HookEventsTestCase) {
```
Comment (norkans7) [inline hooks/hooks_test.go]: Added a way to test events are applied even if the they are not flow actions

Comment (rowanseymour) [reply hooks/hooks_test.go]: Should this be testing modifiers instead? Let them create the events..

Comment (norkans7) [reply hooks/hooks_test.go]: modifiers are as well tested in 

https://github.com/nyaruka/mailroom/pull/270/files#diff-cc2d4d07b0c749193d82b461cf4403d8

this was to make sure we have coverage for the hooks

Comment (rowanseymour) [reply hooks/hooks_test.go]: Sure tho you could make the same argument for the other hook tests which are action based - actions are tested well elsewhere, but by testing via actions, we improve our end to end testing.

For bonus points I'd also merge these two testing mechanisms - i.e. one `HookTestCase` that can take actions or modifiers.

Comment (norkans7) [reply hooks/hooks_test.go]: Okay, I'll update those

Comment (norkans7) [reply hooks/hooks_test.go]: That is updated

**Comment by norkans7** [comment]
Replaced by https://github.com/nyaruka/mailroom/pull/277

**Pull request closed without merging**